### PR TITLE
Rename project

### PR DESCRIPTION
**Pull request by moepman**
These changes resemble the renaming of the proect:
* README.md has been updated to reflect the renaming and also clarify the projects purpose and components
* The package wgtools has been renamed to wgkex (short for wg key exchange)
  * Other options considered were wgke, wgkeyexchange and wg_key_exchange (but have not been chose because undersorces are discouraged in package names, and wgkeyexchange feels a bit lengthy and awkward to read)

The packages / components will be named:
* broker (since it will be more that just a frontend)
* worker: it is consuming the keys and updates the wireguard instance via netlink

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1 +1 @@
-recursive-include wgtools/frontend/templates *
+recursive-include wgke/frontend/templates *
```
Comment (mweinelt) [inline MANIFEST.in:1]: kex is a more common abbreviation of key exchange.

```suggestion
recursive-include wgkex/frontend/templates *
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
-# WireGuard Tools
+# WireGuard Key Exchange
 
-WireGuard Tools consists of both a frontend and a backend to dynamically supply WireGuard keys to a FreiFunk Gateway offering BATMAN over VXLAN over WireGuard.
+WireGuard Key Exchange is a tool consisting of two parts: a frontend (broker) and a backend (daemon). The frontend (broker) is where the client can push (register) its key before connecting. The backend (daemon) is injecting those keys into a WireGuard instance.
```
Comment (mweinelt) [inline README.md:3]: I don't feel daemon to reflects properly what it does.

Comment (awlx) [reply README.md:3]: I agree here but client seems also wrong as it programs the backend, sets routes etc..

Comment (mweinelt) [reply README.md:3]: Worker?

File: MANIFEST.in (hunk 1 above)
Comment (moepman) [reply MANIFEST.in]: Thanks, I've used wgkex in the current draft.

File: README.md (hunk 2 above)
Comment (awlx) [reply README.md:3]: worker sounds good.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
-# WireGuard Tools
+# WireGuard Key Exchange
 
-WireGuard Tools consists of both a frontend and a backend to dynamically supply WireGuard keys to a FreiFunk Gateway offering BATMAN over VXLAN over WireGuard.
+WireGuard Key Exchange is a tool consisting of two parts: a frontend (broker) and a backend (worker). The frontend (broker) is where the client can push (register) its key before connecting. The backend (worker) is injecting those keys into a WireGuard instance.
+This tool is intended to facilitate running BATMAN over VXLAN over WireGuard as a means to create encrypted high-performance mesh links.
```
Comment (mweinelt) [inline README.md:4]: That should probably be opt-in, to keep the tooling generic.

Comment (moepman) [reply README.md:4]: I think that BATMAN o VXLAN o WG is the major usecase, and the initial reason to start this project, so it deserves a spot in the README as to why we are doing this.
In practice the exact behaviour (only inject keys into a matching wg instance, also do some VXLAN related work, etc) could be made configurable?
Unless someone has a concrete replacement for this line I'd let it sit there until we have more actual code.

Comment (mweinelt) [reply README.md:4]: wfm

**Pull request merged**